### PR TITLE
Add nfantoni as a developer for adoption promotion plugin

### PR DESCRIPTION
**Pull request by nfantoni**
# Link to GitHub repository

https://github.com/jenkinsci/artifact-promotion-plugin

# When modifying release permission

List the GitHub usernames of the users who should have commit permissions below:
- @hcguersoy 

This is needed in order to cut releases of the plugin or component.

If you are modifying the release permission of your plugin or component, fill out the following checklist:

```[tasklist]
### Release permission checklist (for submitters)
- [X] The usernames of the users added to the "developers" section in the .yml file are the same the users use to log in to [accounts.jenkins.io](https://accounts.jenkins.io/).
- [x] All users added have logged in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) once.
- [x] I have mentioned an [existing team member](https://github.com/orgs/jenkinsci/teams) of the plugin or component team to approve this request.
```

## When enabling automated releases (cd: true)

Follow the [documentation](https://www.jenkins.io/doc/developer/publishing/releasing-cd/) to ensure, your pull request is set up properly. Don't merge it yet.  
In case of changes requested by the hosting team, an open PR facilitates future reviews, without derailing work across multiple PRs.

### Link to the PR enabling CD in your plugin

```[tasklist]
### CD checklist (for submitters)
- [ ] I have provided a link to the pull request in my plugin, which enables CD according to the documentation. 
```

```[tasklist]
### Reviewer checklist
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```
There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it.

### CODE REVIEW

**Comment by nfantoni** [comment]
I've logged on Jira and Artifactory

**Comment by nfantoni** [comment]
Hi @hcguersoy,

I know and use this plugin, I've contributed in past and I'm interested ti adopt if is possible.

thanks

**Comment by hcguersoy** [comment]
> Hi @hcguersoy,
> 
> I know and use this plugin, I've contributed in past and I'm interested to adopt if is possible.
> 
> thanks

Hey Nicolas, sounds good. 
I would be very happy if you could continue the project.
Firstly I will de-archive the project if possible and add you as a contributor.

**Comment by nfantoni** [comment]
> > Hi @hcguersoy,
> > I know and use this plugin, I've contributed in past and I'm interested to adopt if is possible.
> > thanks
> 
> Hey Nicolas, sounds good. I would be very happy if you could continue the project. Firstly I will de-archive the project if possible and add you as a contributor. Update: de-archive done, but I'm not able to edit the team, so it's upon this PR.

Thanks a lot! I will wait for this PR!

File: permissions/plugin-artifact-promotion.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
 name: "artifact-promotion"
-github: &gh "jenkinsci/artifact-promotion-plugin"
+github: "jenkinsci/artifact-promotion-plugin"
 issues:
-  - github: *gh
+  - github: "https://github.com/jenkinsci/artifact-promotion-plugin/issues"
```
Comment (NotMyFault) [inline permissions/plugin-artifact-promotion.yml:5]: Revert this

**Review by NotMyFault — CHANGES_REQUESTED** [review]

File: permissions/plugin-artifact-promotion.yml (hunk 1 above)
Comment (nfantoni) [reply permissions/plugin-artifact-promotion.yml]: Done

**Pull request merged**